code/datums/holocall.dm

```diff
@@ -173,7 +173,7 @@
 			if(!.)
 				calling_holopad.atom_say("No answer received.")
 				calling_holopad.temp = ""
-
+				qdel(src)
 	else if(!.)
```

Comment:
honestly you could just turn this into a regular if statement and remove the added qdel and have the same functionality

Reply:
Yeah, seems to work all right